<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_widget</class>
 <widget class="QWidget" name="history_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>3</y>
     <width>380</width>
     <height>215</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget:focus {
    outline: none;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_water">
    <attribute name="title">
     <string>水位</string>
    </attribute>
    <widget class="QPushButton" name="button_water_timestart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>4</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>设置查询时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_water">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>7</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="button_water_determine">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>4</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_water">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_flow_speed">
    <attribute name="title">
     <string>流速</string>
    </attribute>
    <widget class="QPushButton" name="button_flow_speed_determine">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>4</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_flow_speed">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>7</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_flow_speed_timestart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>4</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>设置查询时间</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_flow_speed">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_flow_now">
    <attribute name="title">
     <string>瞬时流量</string>
    </attribute>
    <widget class="QPushButton" name="button_flow_now_timestart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>4</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>设置查询时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_flow_now">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>7</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_flow_now_determine">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>4</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_flow_now">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_rain_now">
    <attribute name="title">
     <string>当前雨量</string>
    </attribute>
    <widget class="QTableView" name="tableView_rain_now">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="button_rain_now_timestart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>4</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>设置查询时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_rain_now">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>7</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_rain_now_determine">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>4</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
